src/pages/home/style.js

```diff
@@ -89,4 +89,54 @@ const Anchor = styled.h4`
     margin-top:14px;
     text-align: center;
 `
-export { BodyContainer, Banner, AppName, AppCopy, Container, StyledInput, BlueButton, StyledForm, Anchor }
\ No newline at end of file
+const StyledButtonOptions = styled.div`
+    display: flex;
+    justify-content: center;
+    gap: 30px;
+    margin-top: 20px;
+`
+const StyledDirectLogin = styled.button`
```

Comment:
Aqui o StyledDirectLogin e o StyledCancel parece que só tem uma coisa de diferente, não seria melhor criar um único componente que recebesse o atributo color e dai esse atributo alterasse a cor do botão?

Reply:
Ahh, eu deixei pra alterar isso no final e acabei esquecendo. Agr to em viagem, dai de tarde eu altero e subo dnv, bom q da tempo de mais alguem comentar algo q achar